plugins/colorschemes/modus.nix

```
@@ -0,0 +1,107 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+helpers.neovim-plugin.mkNeovimPlugin config {
+  name = "modus";
+  luaName = "modus-themes";
+  originalName = "modus-themes-nvim";
```

My bad for not spotting this sooner:

```suggestion
  originalName = "modus-themes.nvim";
```